src/main/java/crazypants/enderio/conduit/TileConduitBundle.java

```diff
@@ -791,6 +795,49 @@ public void geometryChanged() {
     serial++;
   }
 
+  private Object node; // IGridNode object, untyped to avoid crash w/o AE2
+
+  @Override
+  @Method(modid = "appliedenergistics2")
+  public IGridNode getGridNode(AEPartLocation loc) {
+    if (loc == null || loc == AEPartLocation.INTERNAL) {
+      return (IGridNode) node;
+    } else {
+      IMEConduit cond = getConduit(IMEConduit.class);
+      if (cond != null) {
```

Comment:
I'm not very familar with EnderIO's code to modify original logic. If your statement is right - I'll remove these checks.